pkg/controller/operators/catalog/operator.go

```diff
@@ -343,8 +343,9 @@ func (o *Operator) syncCatalogSources(obj interface{}) (syncError error) {
 				ResourceVersion: configMap.GetResourceVersion(),
 			}
 			out.Status.LastSync = timeNow()
-
-			// update status
+			if wasOwned := ownerutil.EnsureOwner(configMap, catsrc); !wasOwned {
```

Comment:
Can you add a quick check (just in one of the existing e2e tests) that configmaps get adopted? (I think the ones we generate for testing get created with owner refs, so we can just remove them to test this)

Reply:
Looks like we [already had a test for this](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/test/e2e/catalog_e2e_test.go#L200)?